.github/pull_request_template.md

```diff
@@ -0,0 +1,43 @@
+## Migrations
+
+YES | NO
+
+## Description
+
+A few sentences describing the overall goals of the pull request's commits.
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Improve structure or format of the code(fix lint issues or beautify code syntax)
+- [ ] New tests or updating tests(regarding unit, integration or e2e)
+- [ ] This change requires a documentation update
+
+## Steps to Test or Reproduce
+
+Outline the steps to test or reproduce the PR here.
+
+```sh
+git pull --prune
+git checkout <feature_branch>
+bundle; script/server
+```
+
+## Produced Queries
+
+<!-- Add as sql command produced queries if new queries are added -->
+
```

Comment:
I suggest you adding a fallback script in a comment style, example:

```sql
# there are no new produced or changed queries
```

Reply:
Resolved on https://github.com/DeyvidHolz/nestjs-jwt/pull/1/commits/5677dd267ce2c2d3f02a3998e685babfc97b3a5b